### DISCUSSION TITLE
Rendering a fixed-size point

### DISCUSSION THREAD

**Discussion by ThomasAlban** [bevyengine/bevy#8729] [Q&A]
I'm trying to display a Vector3 as a point in the 3d scene, which, no matter the camera position/zoom, always is displayed the same size. How would I implement this?
Another related question, is it possible to make the point partially viewable through other objects in the scene (as if the material in front of it is semi-transparent)?

**Comment by rlidwka** [comment]
> I'm trying to display a Vector3 as a point in the 3d scene, which, no matter the camera position/zoom, always is displayed the same size. How would I implement this?

The way I'm doing it is adjusting size of that point every frame (which I can calculate based on distance from the camera). It would work for a small number of points. But I'm curious if there's a better solution.
